audit/projects/kubernetes-public/service-accounts/k8s-infra-dns-updater@kubernetes-public.iam.gserviceaccount.com/description.json

```diff
@@ -1,5 +1,5 @@
 {
-  "displayName": "k8s-infra dns updater",
+  "displayName": "k8s-infra-dns-updater",
```

Comment:
Expected from https://github.com/kubernetes/k8s.io/pull/1974

The `displayName` seemed mostly unused, so to reduce the number of parameters to pass during provisioning, I opted to make it the same as `name` (the part before the `@` in the service accounts e-mail address).  More easily searchable within our source code.